]

```diff
@@ -0,0 +1,5 @@
+Terminate batch job (Y/N)? 
```

Comment:
Hrm, I don't think we need any of the updates to any files apart from `crowdin.py` - could you remove this file, the package-lock.json, and the changes to package.json and yarn.lock?

Reply:
Okay yes I would and then I'd push. Thank you.

Reply:
Just did, could you confirm?

Reply:
Looks like there are still a couple of changes to package.json and yarn.lock.

You should be able to restore these to the default state on the develop branch if your develop branch is up to date with the upstream develop branch by doing this:
```
git checkout develop package.json yarn.lock
```
This will update the versions on your branch to the versions on develop - then commit the result so that there is no overall diff in this PR.

Reply:
Alright, I'd look into this. Thanks for the feedback.